src/utils/checkWord.ts

```diff
@@ -0,0 +1,17 @@
+import fs from 'node:fs';
+import readline from 'readline';
+
+export async function isValid(word: string) {
+	const readLine = readline.createInterface({
+		input: fs.createReadStream('src/words/validGuess.txt'),
+		output: process.stdout,
+		terminal: false,
+	});
+
+	let valid = false;
+	for await (const line of readLine) {
+		line === word ? valid = true : null;
```

Comment:
IMO, thernary operator should be used only inside expressions. An if would work here, even if it's more code.

Reply:
Dunno. Is there any way to return the boolean without a variable?

Reply:
I didn't understand your question. But we could make it
```suggestion
		if (line === word) {
			valid = true
		}
```